module/VuFind/src/VuFindTest/Feature/AutoRetryTrait.php

```diff
@@ -107,6 +107,20 @@ public function runBare(): void
                 // Execute callbacks for interrupted test, unless this is the
                 // last round of testing:
                 if ($this->retriesLeft > 0) {
+                    $logMethod = [
+                        $this,
+                        $annotations['method']['retryLogMethod'][0] ?? 'logWarning'
+                    ];
+                    if (is_callable($logMethod)) {
+                        call_user_func(
+                            $logMethod,
+                            'RETRY ' . $this->getName(false)
```

Comment:
Is it worth adding `get_class($this)` in front of `$this->getName()` to be even more specific about where the retry is taking place?

Reply:
Yes, done!